src/omni_epd/displays/waveshare_display.py

```diff
@@ -247,23 +321,18 @@ def close(self):
         epdconfig.module_exit()
 
 
-class WaveshareGrayscaleDisplay(VirtualEPD):
+class Waveshare565finDisplay(VirtualEPD):
     """
-    This class is for the Waveshare displays that support 4 shade grayscale
-
-    https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd2in7.py
-    https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd3in7.py
-    https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd4in2.py
+    This class is for the Waveshare 5.64 in 7 color display
+    https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd5in65f.py
```

Comment:
Waveshare has [another](https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd4in01f.py) 7 color display.

Reply:
Good catch. The link to it in their product listings was bad so I didn't have it in my initial list but after some Googling I did find it. Added now.